atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/keyvalue/impl/RangeRequestTest.java

```diff
@@ -15,41 +15,40 @@
  */
 package com.palantir.atlasdb.keyvalue.impl;
 
-import static org.junit.Assert.assertEquals;
+import static org.assertj.core.api.Assertions.assertThat;
 
 import com.palantir.atlasdb.keyvalue.api.RangeRequest;
 import com.palantir.atlasdb.keyvalue.api.RangeRequests;
-import org.junit.Assert;
 import org.junit.Test;
 
 public class RangeRequestTest {
 
     @Test
     public void testPrefix() {
         byte[] end = RangeRequest.builder().prefixRange(new byte[] {-1}).build().getEndExclusive();
-        assertEquals(0, end.length);
+        assertThat(end.length).isEqualTo(0);
         end = RangeRequest.builder().prefixRange(new byte[] {-2}).build().getEndExclusive();
-        assertEquals(1, end.length);
-        assertEquals(-1, end[0]);
+        assertThat(end.length).isEqualTo(1);
+        assertThat(end[0]).isEqualTo(-1);
```

Comment:
@carterkozak Probably don't need to fix this, but amazingly this _isn't_ semantically equivalent! (The original works because `Assert#assertEquals` looks like it does widening conversions to longs, but this fails because the byte `-1` and int `-1` aren't considered equal in AssertJ, probably because they are different types after unboxing).

Reply:
Yep, we definitely handled some of these, but I must have neglected to implement byte/int conversion. I think I used the rules for java implicit conversion, which doesn't quite match assertj behavior. Mind filing a ticket on https://github.com/palantir/assertj-automation ?

Reply:
Yep, here goes: https://github.com/palantir/assertj-automation/issues/261 and a quick scan of the code suggests that yeah, it goes to the catchall Object rules for these